ResourceManager.rb

```diff
@@ -0,0 +1,11 @@
+class ResourceManager
+    def initialize
+        # True indicates that the resource is available
+        @scanner = True
+        @printer_1 = True
+        @printer_2 = True
+        @modem = True
+        @sata_1 = True
+        @sata_2 = True
+    end
+end
```

Comment:
To avoid comments and also bring some context, I think we can refactor to:

```ruby
class ResourceManager
  AVAILABLE = 'available'
  DISABLED = 'disabled'

    def initialize
        @scanner = AVAILABLE
        @printer_1 = AVAILABLE
        @printer_2 = AVAILABLE
        @modem = AVAILABLE
        @sata_1 = AVAILABLE
        @sata_2 = AVAILABLE
    end
end
```